### PR TITLE
Updated readme - still more work to do but wanted to see what it looks like for now

### PR DESCRIPTION
**Pull request by manassaini**

### CODE REVIEW

**Comment by manassaini** [comment]
Unable to add the package "uvloop" in PyCharm. There is no support for uvloop in Windows. This causing errors within Sphinx and not generating the documentation files correctly.

File: README.md [hunk 1]
```diff
@@ -14,13 +13,87 @@ gives the end users an arduino-eqsue interface, and provides movement and utilit
 ![Integration Tests](https://github.com/AutonoLab/pteranodon/actions/workflows/integration-tests.yaml/badge.svg?branch=main)
 
 ## Contributing
+Information TBD
```
Comment (justincdavis) [inline README.md:16]: Contributing should be its own .md file CONTRIBUTING.md

File: README.md [hunk 2]
```diff
@@ -14,13 +13,87 @@ gives the end users an arduino-eqsue interface, and provides movement and utilit
...
 [MyPy](https://mypy.readthedocs.io/en/stable/).
 
 In order to merge into the `main` branch, a contributor must satisfy these standards as well as the
 [Black Code Style](https://black.readthedocs.io/en/stable/).
 
-This can be done automatically by setting your IDE to format on save, or you can run `black --safe ./pteranodon` locally.
\ No newline at end of file
+This can be done automatically by setting your IDE to format on save, or you can run `black --safe ./pteranodon` locally.
+
+## Description
```
Comment (justincdavis) [inline README.md:28]: Remove same description that sits higher up

File: README.md [hunk 3]
```diff
@@ -14,13 +13,87 @@ gives the end users an arduino-eqsue interface, and provides movement and utilit
...
+   * `exit`
+   * `wsl --shutdown`
+   * `wsl`
+2. Switch to the PX4 repository in the WSL home folder:
+   * `cd ~/PX4-Autopilot`
+3. Build the PX4 SITL target and test your environment:
+   * `make px4_sitl`
+
+### Using Docker
+
```
Comment (justincdavis) [inline README.md:91]: Add steps for our docker container (this is waiting on the docker from @david-shearon )

**Review by justincdavis — CHANGES_REQUESTED** [review]
Missing generated .rst files for actual implementations?

Refer to comments on README.md about requested changesd

**Comment by manassaini** [comment]
> Missing generated .rst files for actual implementations?
> 
> Refer to comments on README.md about requested changesd

**Comment by manassaini** [comment]
> 

I believe those are not needed. Only for directories. It is generating correct .html files for plugins and everything so should be fine

**Pull request closed without merging**